backend/file-system/src/main/java/com/htec/filesystem/controller/FolderController.java

```diff
@@ -51,4 +54,20 @@ public ResponseEntity createFolder(@RequestBody CreateFolderRequestModel createF
 
         return ResponseEntity.status(retStatus).body(new TextResponseMessage("Folder created", retStatus.value()));
     }
+
+    @PutMapping("/move-to-trash")
```

Comment:
same as previous PR, take a look at the PATCH request

Reply:
**java.net.ProtocolException: Invalid HTTP method: PATCH**
	at java.base/java.net.HttpURLConnection.setRequestMethod(HttpURLConnection.java:487) ~[na:na]
	at java.base/sun.net.www.protocol.http.HttpURLConnection.setRequestMethod(HttpURLConnection.java:570) ~[na:na]
	at org.springframework.http.client.SimpleClientHttpRequestFactory.prepareConnection(SimpleClientHttpRequestFactory.java:217) ~[spring-web-5.3.12.jar:5.3.12]
	at org.springframework.http.client.SimpleClientHttpRequestFactory.createRequest(SimpleClientHttpRequestFactory.java:146) ~[spring-web-5.3.12.jar:5.3.12]
	at org.springframework.http.client.support.HttpAccessor.createRequest(HttpAccessor.java:124) ~[spring-web-5.3.12.jar:5.3.12]
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:772) ~[spring-web-5.3.12.jar:5.3.12]
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:711) ~[spring-web-5.3.12.jar:5.3.12]
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:602) ~[spring-web-5.3.12.jar:5.3.12]
	**at com.htec.gateway.service.GatewayService.send(GatewayService.java:117)** ~[classes/:na]

Reply:
Changed RestTemplate requestFactory to HttpComponentsClientHttpRequestFactory. Now it passes PATCH requests